code/game/objects/items/supermatter_delaminator.dm

```diff
@@ -0,0 +1,213 @@
+	return FALSE
+
+/obj/item/supermatter_delaminator/antinoblium_shard/attackby(obj/item/W, mob/living/user, params)
+	if(istype(W, /obj/item/hemostat/antinoblium))
+		var/obj/item/hemostat/antinoblium/tongs = W
+		if (tongs.shard)
+			to_chat(user, "<span class='notice'>\The [tongs] is already holding an antinoblium shard!</span>")
+			return FALSE
+		forceMove(tongs)
+		tongs.shard = src
+		tongs.update_icon()
+		to_chat(user, "<span class='notice'>You carefully pick up [src] with [tongs].</span>")
+	else if(istype(W, /obj/item/scalpel/supermatter) || istype(W, /obj/item/nuke_core_container/supermatter/)) // we don't want it to dust
+		return
+	else
+		to_chat(user, "<span class='notice'>As it touches \the [src], both \the [src] and \the [W] burst into dust!</span>")
+		radiation_pulse(user, 100)
+		playsound(src, 'sound/effects/supermatter.ogg', 50, 1)
+		qdel(W)
+		qdel(src)
+
+/obj/item/supermatter_delaminator/antinoblium_shard/pickup(mob/living/user)
+	..()
+	if(!iscarbon(user))
+		return FALSE
+	var/mob/ded = user
+	user.visible_message("<span class='danger'>[ded] reaches out and tries to pick up [src]. [ded.p_their()] body starts to glow and bursts into flames before flashing into dust!</span>",\
+			"<span class='userdanger'>You reach for [src] with your hands. That was dumb.</span>",\
+			"<span class='italics'>Everything suddenly goes silent.</span>")
+	radiation_pulse(user, 500, 2)
+	playsound(get_turf(user), 'sound/effects/supermatter.ogg', 50, 1)
+	ded.dust()
+
+/obj/item/antinoblium_container
+	name = "antinoblium bin"
+	desc = "A small cube that houses a stable antinoblium shard  to be safely stored."
+	icon = 'icons/obj/supermatter_delaminator.dmi'
+	icon_state = "antinoblium_container_sealed"
+	//item_state = "tile"
+	//lefthand_file = 'icons/mob/inhands/misc/sheets_lefthand.dmi'
+	//righthand_file = 'icons/mob/inhands/misc/sheets_righthand.dmi'
+	var/obj/item/supermatter_delaminator/antinoblium_shard/shard
+	var/sealed = TRUE
+
+/obj/item/antinoblium_container/Initialize()
+	. = ..()
+	shard = new /obj/item/supermatter_delaminator/antinoblium_shard
+
+/obj/item/antinoblium_container/Destroy()
+	QDEL_NULL(shard)
+	return ..()
+
+/obj/item/antinoblium_container/proc/load(obj/item/hemostat/antinoblium/T, mob/user)
+	if(!istype(T) || !T.shard || shard || sealed)
+		return FALSE
+	T.shard.forceMove(src)
+	shard = T.shard
+	T.shard = null
+	T.update_icon()
+	update_icon()
+	to_chat(user, "<span class='warning'>Container is resealing...</span>")
+	addtimer(CALLBACK(src, .proc/seal), 50)
+	return TRUE
+
+/obj/item/antinoblium_container/proc/unload(obj/item/hemostat/antinoblium/T, mob/user)
+	if(!istype(T) || T.shard || !shard || sealed)
+		return FALSE
+	src.shard.forceMove(T)
+	T.shard = shard
+	shard = null
+	T.update_icon()
+	update_icon()
+	to_chat(user, "<span class='warning'>[user] gingerly takes out the antinoblium shard with the tongs...</span>")
```

Comment:
You are using to_chat here, if this is intended to be a stealthy action, replace user with "You". If it is not, use visible_message()

Reply:
Not a stealthy action, but good to know for when I need it. Thanks!